L1Trigger/L1TMuonEndCap/interface/bdt/LossFunctions.h

```diff
@@ -139,8 +139,8 @@ class Huber : public LossFunction
         double target(Event* e) override
         {
         // The gradient of the loss function.
-
-            if (std::abs(e->trueValue - e->predictedValue) <= quantile)
+	  double multiplier = ((e->trueValue - e->predictedValue) > 0) ? 1.0 : -1.0;
+	  if ((e->trueValue - e->predictedValue)*multiplier <= quantile)
```

Comment:
What is the advantage over std::abs?

Reply:
@mrodozov , I do not understand the reason behind this change.

Reply:
the compiler was complaining with an error not to use abs on a double. so I substituted the usage of abs with an multiplier